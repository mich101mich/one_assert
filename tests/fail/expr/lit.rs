fn main() {
    one_assert::assert!("true");
    one_assert::assert!(1);
    one_assert::assert!(1u8);
    one_assert::assert!(0x01u8);
}
