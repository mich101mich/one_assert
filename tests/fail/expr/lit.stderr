error[E0308]: mismatched types
 --> tests/fail/expr/lit.rs:2:25
  |
2 |     one_assert::assert!("true");
  |                         ^^^^^^ expected `bool`, found `&str`

error[E0308]: mismatched types
 --> tests/fail/expr/lit.rs:3:25
  |
3 |     one_assert::assert!(1);
  |                         ^ expected `bool`, found integer

error[E0308]: mismatched types
 --> tests/fail/expr/lit.rs:4:25
  |
4 |     one_assert::assert!(1u8);
  |                         ^^^ expected `bool`, found `u8`

error[E0308]: mismatched types
 --> tests/fail/expr/lit.rs:5:25
  |
5 |     one_assert::assert!(0x01u8);
  |                         ^^^^^^ expected `bool`, found `u8`
